sunpy/map/tests/test_map_factory.py

```diff
@@ -47,195 +42,197 @@ def test_two_map_inputs(args1, args2):
-        maps_sorted = [sunpy.map.Map(os.fspath(f)) for f in files_sorted]
-        assert all([m.date == m_s.date for m, m_s in zip(maps, maps_sorted)])
-
-        # Pathlib
-        path = pathlib.Path(AIA_171_IMAGE)
-        aiamap = sunpy.map.Map(path)
-        assert isinstance(aiamap, sunpy.map.GenericMap)
-        maps = sunpy.map.Map(eit_fits_directory)
-        assert isinstance(maps, list)
-        assert ([isinstance(amap, sunpy.map.GenericMap) for amap in maps])
-
-        # Glob
-        pattern = os.path.join(eit_fits_directory, "*")
-        maps = sunpy.map.Map(pattern)
-        assert isinstance(maps, list)
-        assert ([isinstance(amap, sunpy.map.GenericMap) for amap in maps])
-        # Test that returned maps are sorted
-        files_sorted = sorted(list(pathlib.Path(pattern).parent.glob('*')))
-        maps_sorted = [sunpy.map.Map(os.fspath(f)) for f in files_sorted]
-        assert all([m.date == m_s.date for m, m_s in zip(maps, maps_sorted)])
-        # Single character wildcard (?)
-        pattern = os.path.join(eit_fits_directory, "efz20040301.0?0010_s.fits")
-        maps = sunpy.map.Map(pattern)
-        assert isinstance(maps, list)
-        assert len(maps) == 7
-        assert ([isinstance(amap, sunpy.map.GenericMap) for amap in maps])
-        # Character ranges
-        pattern = os.path.join(eit_fits_directory, "efz20040301.0[2-6]0010_s.fits")
-        maps = sunpy.map.Map(pattern)
-        assert isinstance(maps, list)
-        assert len(maps) == 4
-        assert ([isinstance(amap, sunpy.map.GenericMap) for amap in maps])
-
-        # Already a Map
-        amap = sunpy.map.Map(maps[0])
-        assert isinstance(amap, sunpy.map.GenericMap)
-
-        # A list of filenames
-        maps = sunpy.map.Map(list(eit_fits_directory.glob('*.fits')))
-        assert isinstance(maps, list)
-        assert ([isinstance(amap, sunpy.map.GenericMap) for amap in maps])
-
-        # Data-header pair in a tuple
-        pair_map = sunpy.map.Map((amap.data, amap.meta))
-        assert isinstance(pair_map, sunpy.map.GenericMap)
-
-        # Data-header pair not in a tuple
-        pair_map = sunpy.map.Map(amap.data, amap.meta)
-        assert isinstance(pair_map, sunpy.map.GenericMap)
-
-        # Data-wcs object pair in tuple
-        pair_map = sunpy.map.Map((amap.data, WCS(AIA_171_IMAGE)))
-        assert isinstance(pair_map, sunpy.map.GenericMap)
-
-        # Data-wcs object pair not in a tuple
-        pair_map = sunpy.map.Map(amap.data, WCS(AIA_171_IMAGE))
-        assert isinstance(pair_map, sunpy.map.GenericMap)
-
-        # Data-header from FITS
-        with fits.open(AIA_171_IMAGE) as hdul:
-            data = hdul[0].data
-            header = hdul[0].header
-        pair_map = sunpy.map.Map((data, header))
-        assert isinstance(pair_map, sunpy.map.GenericMap)
-        pair_map, pair_map = sunpy.map.Map(((data, header), (data, header)))
-        assert isinstance(pair_map, sunpy.map.GenericMap)
+
```

Comment:
I haven't bothered checking this big chunk of diff, but trust that it's just been de-indented without change.

Reply:
That is all I did, de-indent + remove `self` from function arguments.